lib/dor/text_extraction/speech_to_text.rb

```diff
@@ -52,10 +52,50 @@ def cleanup_s3_folder
       # return a list of filenames that should be stt'd
       # iterate over all files in cocina_object.structural.contains, looking at mimetypes
       # return a list of filenames that are correct mimetype
+      # then filter out any files that either (1) do not have an audio track or (2) have audio that is mostly silent
       def filenames_to_stt
-        stt_files.map(&:filename)
+        available_files = stt_files.map(&:filename)
+        available_files.select { |filename| has_audio_track?(filename) && has_useful_audio_track?(filename) }
       end
 
+      # determines if the given file has an audio track by parsing the technical metadata and looking at the file's audio_count
+      def has_audio_track?(filename)
+        file_level_tech_metadata(filename)&.dig('av_metadata', 'audio_count')&.positive? || false
+      end
+
+      # using new technical metadata generated in https://github.com/sul-dlss/technical-metadata-service/pull/572
+      # this uses the audio max_volume and mean_volume fields to determine if the audio is mostly silent
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
+      def has_useful_audio_track?(filename)
+        audio_metadata = file_level_tech_metadata(filename)&.dig('dro_file_parts')&.find { |parts| parts['part_type'] == 'audio' }&.dig('audio_metadata')
+        return false unless audio_metadata && audio_metadata['max_volume'] && audio_metadata['mean_volume']
```

Comment:
note that this "return false" means that if the audio metadata with max/mean volume is NOT present, it will not send it to speech to text.  this means any existing media objects won't work unless they have technical metadata re-generated.

we could be more inclusive here and instead return true instead of false in that situation ...